SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/GridBasedLBStrategy/TestGridBasedLBStrategy.cpp

```diff
@@ -15,5 +15,7 @@ UGridBasedLBStrategy* UTestGridBasedLBStrategy::Create(uint32 InRows, uint32 InC
 
 	Strat->InterestBorder = InterestBorder;
 
+	Strat->AddToRoot();
```

Comment:
I'd rather not use root where possible - can't we just make the start a UPROPERTY member var?

Reply:
I don't think so, no - this class is used in the GridBasedLBStrategy test, so Unreal won't be able to track references to it. I should probably move AddToRoot there.

Reply:
Is the flow different there? My understanding is that the GridBasedLBStrategy should be created as a NewObject regardless, so why wouldn't Unreal GC be aware of it?